### DISCUSSION TITLE
is there Schematics for Sandwich Version?

### DISCUSSION THREAD

**Discussion by armdan** [adonno/tagreader#129] [General]
Hi, I was wondering if you would share Schematics for Sandwich Version - without D1 mini

thanks

**Comment by half5cat** [comment]
I am interested aswell.

**Comment by Patrick1967** [comment]
I too am interested.  Thank you.

**Comment by smic717394** [comment]
I think its ESP8266 v4 since its ubc,  rest is the same except for the middle board.
